src/components/FilePreview/FilePreview.scss

```diff
@@ -0,0 +1,102 @@
+@use '../variables';
+
+$block: '.#{variables.$ns}file-preview';
+
+#{$block} {
+    position: relative;
+
+    width: 120px;
+
+    &:hover,
+    &:focus-within {
+        #{$block}__actions {
+            opacity: 1;
+        }
+    }
+
+    &__actions {
+        position: absolute;
+        top: -10px;
+        right: -10px;
+        z-index: 10;
+
+        display: flex;
+        gap: 4px;
+
+        opacity: 0;
+    }
+
+    &__card {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+
+        padding: 4px 10px;
+
+        &-icon {
```

Comment:
I'm personally prefer not to split `Element` name via `&` (OK if it is `Modificator`), but that not a hard rule